File: client/src/app/estimate/models/item.ts
import { _2d } from "./_2d";
import { _3d } from "./_3d";

export class item {
    public name = "";
    public _2d_arr: [_2d] = [new _2d()];
    public _3d_arr: [_3d] = [new _3d()];
    public area = 0;
    constructor() { }
}

File: server/src/main/java/com/airtajal/model/item.java
package com.airtajal.model;

import java.util.Set;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AccessLevel;
import lombok.Data;
import lombok.experimental.FieldDefaults;
import java.io.Serializable;

@Data
@FieldDefaults(level = AccessLevel.PRIVATE)
@Entity
public class item implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Long id;

    String name;
    Integer area;
    Set<_2d> _2d_arr;
    Set<_3d> _3d_arr;

}
